src/contrib/php_array_api.h

```diff
@@ -26,13 +26,13 @@
 #include "zend_list.h"
 
 #ifdef ZEND_ENGINE_3
-# define PAA_LENGTH_ADJ(l) (l)
-# define PAA_SYM_EXISTS zend_symtable_str_exists
-# define PAA_SYM_DEL    zend_symtable_str_del
+#define PAA_LENGTH_ADJ(l) (l)
```

Comment:
Can we exclude files in `contrib/`, for the same reason we don't want to process things in `src/libmongoc`, etc.?